src/cloudant/account.py

```diff
@@ -61,16 +61,30 @@ def __init__(self, user, auth_token, **kwargs):
         self.encoder = kwargs.get('encoder') or json.JSONEncoder
         self.r_session = None
 
+    @property
+    def admin_party(self):
+        """
+        If neither a username or a password have been provided during object
+        construction then CouchDB Admin Party mode is assumed.
+
+        :returns: Boolean value showing whether a username and password have
+            been provided
+        """
+        if self._cloudant_user is None and self._cloudant_token is None:
```

Comment:
so what happens if the user just fails at passing user/auth_token here? It will assume the db is in admin party mode, which it may be but it may not be. 
I think perhaps the ctor args would be better done by still requiring user and auth_token and adding an admin_party=False. That way the user is clear about setting admin_party and makes it a special case, rather than making is possible for a new user to do the wrong thing because they arent passing any arguments. 

If the user=None, auth_token=None, admin_party=True is too ugly (yeah, its pretty gross) add a helper context:
```python
with couchdb_admin_party(args here):
     # do stuff
```